neo4j/values.yaml

```diff
@@ -266,14 +266,20 @@ config:
 #  apoc.trigger.enabled: "true"
 #  apoc.jdbc.apoctest.url: "jdbc:foo:bar"
 
-# securityContext defines privilege and access control settings for a Pod or Container. Making sure that we dont run Neo4j as root user.
+# securityContext defines privilege and access control settings for a Pod. Making sure that we dont run Neo4j as root user.
 securityContext:
```

Comment:
should this be Pod to make it clearer?

Reply:
This will become a breaking change hence we cannot rename it until version 6

Reply:
yeah that is fine